src/main/msp/msp_box.h

```diff
@@ -23,8 +23,8 @@
 #include "fc/rc_modes.h"
 
 typedef struct box_s {
-    const uint8_t boxId;            // see boxId_e
     const char *boxName;            // GUI-readable box name
+    const uint8_t boxId;            // see boxId_e
```

Comment:
what's the reason for changing this?

Reply:
It helps with struct size. Check `sizeof(box_t)` for old struct which is 12B due to alignment and then new struct which is `8B` so we can save 4B per each instance just by reordering.

Rule of thumb: sort it by size from big to small (or vice versa).